sdk/lib/async/future.dart

```diff
@@ -773,6 +845,27 @@ abstract class Future<T> {
   ///
   /// If `onTimeout` is omitted, a timeout will cause the returned future to
   /// complete with a [TimeoutException].
+  ///
+  /// Example:
+  /// ```dart
+  /// void main() async {
+  ///   var result = await Future.value(waitTask()
+  ///       .timeout(const Duration(seconds: 10)));
+  ///   print(result); // 'completed'
+  ///
+  ///   result = await Future.value(waitTask()
+  ///       .timeout(const Duration(seconds: 1), onTimeout: () => 'timeout'));
+  ///   print(result); // 'timeout'
+  ///
+  ///   result = await Future.value(waitTask()
+  ///       .timeout(const Duration(seconds: 2))); // Throws.
```

Comment:
`Throws` -> `Throws TimeoutException` ?

Reply:
Probably no need to specify error because it is also said on details before example code block.